db/types/alteration.py

```diff
@@ -87,6 +93,31 @@ def alter_column_type(
         conn.execute(DDL(alter_stmt))
 
 
+def get_column_cast_expression(column, target_type_str, engine):
```

Comment:
I really appreciate `str` in the variable name here. Makes this much easier to read.